node/alphanet/src/lib.rs

```diff
@@ -350,4 +364,49 @@ mod tests {
         wait(|| charlie.client.shard_client.chain.best_index() >= 2, 500, 60000);
         wait(|| dan.client.shard_client.chain.best_index() >= 2, 500, 60000);
     }
+
+    #[test]
+    /// One node produces 500 blocks and the other node starts and tries to catch up.
```

Comment:
do 50? and fetch in 10?

Reply:
I don't think I can change the chunk size here due to how everything is glued together.